utils/submit_tracks.py

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python3
+
+import sys
+import json
+import requests
+
+track_api_root = 'https://staging-2020.ensembl.org/api/tracks'
```

Comment:
This shouldn't be exposed. Read it from environment or pass it as argument.

Reply:
Can you elaborate what is the downside of including staging URL in an utility script? Our repos already expose internal URLs in various files (e.g. k8s and ci config files).

Reply:
But I agree that setting the target endpoint URL in code is not ideal. I'll replace it with input arg.

Reply:
Done.